app/services/foreman/deprecation.rb

```diff
@@ -11,7 +11,7 @@ def self.check_version_format(foreman_version_deadline)
     end
 
     def self.api_deprecation_warning(info)
-      ActiveSupport::Deprecation.warn("Your API call uses deprecated behavior, #{info}", caller)
+      Rails.logger.warn "DEPRECATION WARNING: #{info}"
```

Comment:
should there be any `caller` information showing a stacktrace logged? This would be helpful for us to find the deprecation and was the existing behavior

Reply:
Please no, playing around with `caller` is slow as hell.